tests/unit/test_routes/test_resource_update.py

```diff
@@ -1,3 +1,4 @@
+from tests.unit.test_response_util import client
```

Comment:
Is this line necessary? I think later on in the file `client` is going to refer to `module_client` which is passed in by pytest, so this is actually an unused import, right?

Reply:
yeah it is indeed unused